test/tests.py

```diff
@@ -305,3 +310,34 @@ def test_0006_OSB(self):
             if source_count != target_count:
                 self.assertEqual(source_count, target_count, f'{index}: doc counts do not match - '
                                                              f'Source = {source_count}, Target = {target_count}')
+
+    def test_0007_timeBetweenRequestsOnSameConnection(self):
+        # This test will verify that the replayer functions correctly when
+        # requests on the same connection on the proxy that has a minute gap
+        seconds_between_requests = 60  # 1 minute
+
+        # Set a consistent session to be used for this test
+        session = Session()
+        adapter = HTTPAdapter(pool_connections=1, pool_maxsize=1, max_retries=1)
+        session.mount('http://', adapter)
+        session.mount('https://', adapter)
```

Comment:
why are you using both of these?

Reply:
to work in both ssl and non-ssl cases

Reply:
What does mount do?

Reply:
mount applies the adapter to a given uri prefix. I've since modified the code to directly use the proxy endpoint for clarity here